packages/view/src/Charcoal/View/Twig/UrlHelpers.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Charcoal\View\Twig;
+
+// From 'charcoal-view'
+use Charcoal\View\Twig\HelpersInterface;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig helpers for Url.
+ */
+class UrlHelpers extends AbstractExtension
+    implements HelpersInterface
+{
+    /**
+     * @param array $data Class Dependencies.
+     */
+    public function __construct(array $data = null)
+    {
+        if (isset($data['baseUrl'])) {
+            $this->baseUrl = $data['baseUrl'];
+        }
+    }
+
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('baseUrl', [$this, 'baseUrl']),
+            new TwigFunction('siteUrl', [$this, 'baseUrl']),
+            new TwigFunction('withBaseUrl', [$this, 'withBaseUrl']),
+        ];
+    }
+
+    /**
+     * Render the Twig baseUrl function.
+     *
+     * @return string
+     */
+    public function baseUrl()
+    {
+        if (null === $this->baseUrl) {
+            return '';
+        }
+
+        return $this->baseUrl;
```

Comment:
Simplify with null coalescing operator (`??`).

```suggestion
        return $this->baseUrl ?? '';
```